src/activy/utils/getControl.py

```
@@ -0,0 +1,57 @@
+    Returns the center coordinates of the expected control if found, else None.
+    """
+    # Zorg ervoor dat expectedControl een lijst is
+    if isinstance(expectedControl, (list, tuple)):
+        expectedControlList = [str(item) for item in expectedControl]
+    else:
+        expectedControlList = [str(expectedControl)]
+
+    if debug:
+        screenshotPath = "screen.png"
+    else:
+        screenshotPath = tempfile.mktemp(suffix=".png")
```

Ruff (0.8.2)

Use ternary operator `screenshotPath = "screen.png" if debug else tempfile.mktemp(suffix=".png")` instead of `if`-`else`-block

Replace `if`-`else`-block with `screenshotPath = "screen.png" if debug else tempfile.mktemp(suffix=".png")`

(SIM108)